cmd/installations/create.go

```diff
@@ -181,7 +184,18 @@ func (o *createOpts) run(ctx context.Context, cmd *cobra.Command, log logr.Logge
 		}
 	}
 
-	cmd.Println(string(marshaledYaml))
+	if o.outputPath == "" {
+		cmd.Println(string(marshaledYaml))
+	} else {
+		f, err := os.Create(o.outputPath)
+		if err != nil {
+			return fmt.Errorf("Error creating file %s:%w", o.outputPath, err)
+		}
+		_, err = f.Write(marshaledYaml)
+		if err != nil {
+			return fmt.Errorf("Error writing file %s:%w", o.outputPath, err)
```

Comment:
Just a small thing: I would suggest to not use capitalization when wrapping errors.